Resources/Maps/_NF/Dungeon/cave_factory.yml

```diff
@@ -6758,6 +6783,21 @@ entities:
     - type: Transform
       pos: 25.5,12.5
       parent: 1
+  - uid: 1943
+    components:
+    - type: Transform
+      pos: 33.5,19.5
+      parent: 1
+  - uid: 1944
+    components:
+    - type: Transform
+      pos: 32.5,19.5
+      parent: 1
+  - uid: 1945
+    components:
+    - type: Transform
+      pos: 31.5,19.5
+      parent: 1
```

Comment:
Game doesn't seem to like these entities, their IDs are already in use for engineering loot spawn.

Reply:
Might have fucked something with the grid save, I can fix in few hours.